capitalize_decorator.rb

```diff
@@ -0,0 +1,6 @@
+# Defines a capitalize_decorator class
+class CapitalizeDecorator < BaseDecorator
```

Comment:
Same here, please require the `BaseDecorator` class from the `base_decorator.rb` file using `require_relative` to avoid the **NameError**.

Reply:
Implemented